src/platform/testing/e2e/cypress/plugins/index.js

```diff
@@ -75,4 +78,5 @@ module.exports = async on => {
     table: message => console.log(table(message, tableConfig)) || null,
     /* eslint-enable no-console */
   });
+  return config;
```

Comment:
Maybe put a nice lil empty line before the `return` statement?

Reply:
Sure thing!